docs/app/pages/css/css-sections/icons/icons.module.ts

```diff
@@ -16,6 +16,7 @@ import { CssUxIconsComponent } from './ux-icons/ux-icons.component';
 import { IconModule, TabsetModule, TooltipModule } from '@ux-aspects/ux-aspects';
 import { IconPreviewComponent } from './ux-icons/icon-preview/icon-preview.component';
 import { IconSnippetComponent } from './ux-icons/icon-preview/icon-snippet/icon-snippet.component';
+import { ButtonsModule } from 'ngx-bootstrap';
```

Comment:
We should update this to import path:

```typescript
import { ButtonsModule } from 'ngx-bootstrap/buttons';
```

Importing directly from `ngx-bootstrap` imports the entire `ngx-bootstrap` library, and while a lot of it can be tree shaken away, there are some parts which cannot safely be removed by Webpack. So we can import directly from buttons to avoid unnecessary code being bundled.

As you can see the difference in the size of code being bundled even after tree shaking:
![image](https://user-images.githubusercontent.com/20795331/62871760-fab98000-bd13-11e9-8f1d-f125d1167925.png)